test/fixture/long-stack-trace/test.js

```diff
@@ -0,0 +1,14 @@
+import test from '../../../';
+const Promise = require('./enable-trace.js');
+
+test(async t => {
+	const resolve = await Promise.resolve().then(function outer() {
```

Comment:
Would be good to add a comment linking to http://bluebirdjs.com/docs/api/promise.longstacktraces.html, since that's where the code comes from.